src/core/ucc_global_opts.c

```diff
@@ -30,6 +30,13 @@ ucc_config_field_t ucc_global_config_table[] = {
      ucc_offsetof(ucc_global_config_t, log_component),
      UCC_CONFIG_TYPE_LOG_COMP},
 
+    {"COLL_TRACE", "n",
+     "If UCC logging level is INFO or higher UCC will print info message"
```

Comment:
a bit confusing since you're explaining what would happen in case of "y" but defining as "n"

Reply:
why is it confusing? by default coll trace is disabled because it produces a lot of output. If you want to see it just set UCC_COLL_TRACE=y.

Reply:
In my opinion it would be more clear like this:
"If enabled and UCC logging level is INFO or higher UCC will print info message per collective.\n"